qa/suites/fs/basic_functional/tasks/mds-full.yaml

```diff
@@ -8,10 +8,15 @@ overrides:
       - failsafe disengaged, no longer dropping
       - is full \(reached quota
     conf:
+      mon:
+        mon osd nearfull ratio: 0.6
+        mon osd backfillfull ratio: 0.6
+        mon osd full ratio: 0.7
       osd:
         osd mon report interval max: 5
         osd objectstore: memstore
-        memstore device bytes: 100000000
+        osd failsafe full ratio: 1.0
```

Comment:
If the test was failing because of OSDs hitting the failsafe full threshold, then shouldn't this `osd failsafe full ratio: 1.0` fix the test?  Why are the other adjustments to nearfull etc needed?